modules/ROOT/pages/dataweave-language-introduction.adoc

```diff
@@ -399,6 +399,30 @@ In the Mule app XML, you can use the `${file::filename}` syntax to send a script
 </choice>
 ----
 
+[[dw_readers_writers]]
+== DataWeave Readers and Writers
+
+DataWeave can read input data as a whole in-memory, in indexed fashion,
+and for some data formats, part-by-part by streaming the input. When attempting
+to read a large file, it is possible to run out of memory or to impact
+performance negatively. Streaming can improve performance but impacts access to file.
```

Comment:
How does streaming impact access to file? Is it that it impacts **access speed to files**?